packages/xchain-bitcoin/__mocks__/block-chair.ts

```diff
@@ -0,0 +1,50 @@
+import axios from 'axios'
+import MockAdapter from 'axios-mock-adapter'
+import { addresses, rawTransactions, dashboardTransactions, bitcoinStats } from './responses.json'
+
+const mock = new MockAdapter(axios)
+
+type MockConfig = {
+  url?: string
+}
+
+export default {
+  restore: mock.restore,
+  init: () => {
+    //Mock getAddress
+    mock.onGet(/\/dashboards\/address\//).reply(function (config: MockConfig) {
+      const id: any = config.url?.split('/').pop()
+      return [200, { data: { [id]: (addresses as any)[id] } }]
```

Comment:
Same here. Can't types of [blockchair-api-types.ts](https://github.com/xchainjs/xchainjs-lib/blob/8b6ad2eec2e4ebf142d3dc86667c167403024778/packages/xchain-bitcoin/src/types/blockchair-api-types.ts) used here?

Reply:
Fixed